book/P1C6_survtsk.qmd

```diff
@@ -10,119 +10,270 @@ abstract: "This chapter introduces the different survival problems and formalise
+This only reaches zero if every individual at risk at the last observed time-point experiences the event: $d_{t_{(k)}} = n_{t_{(k)}}$.
+In practice, due to administrative censoring, there will almost always be censoring at the final time-point.
+As a result, $d_{t_{(k)}} < n_{t_{(k)}}$ and $\hatS(\infty) > 0$.
+Heuristics have been proposed to address this including linear extrapolation to zero, or dropping the curve to zero at the final time-point, however these can introduce significant bias in the estimated survival time [@han.restricted.2022, @Sonabend2022].
+Another possibility is to instead report the median survival time, but this is only defined if the survival curves drop below $0.5$ within the observed period, which is not guaranteed [@Haider2020].
 
-Predicting survival times is the *deterministic survival task*, defined by predicting a continuous value in the positive Reals and is specified by $g: \calX \rightarrow \NNReals$.
-See @sec-conclusions-faq for practical discussion around predicting in $\NNReals$ vs. $\PReals$ and continuous vs discrete time representations.
-Formally, whilst this is a special case of the ranking task with $\calR \subseteq \NNReals$, the distinction is important as a 'deterministic' prediction specifically refers to forecasting a single determined outcome with a meaningful interpretation, whereas the 'ranking' task is not a deterministic forecast of an event.
+One alternative is to estimate the *restricted mean survival time* (RMST) [@han.restricted.2022; @andersen.regression.2004].
+In contrast to @eq-darth, which integrates the survival curve over the entire time axis, the RMST places an upper-bound on the integral:
 
-<!-- FIXME add example -->
+$$
+\RMST(\tau) = \int^\tau_0 S_Y(y) \ \dy
+$$ {#eq-rmst}
+
+Clearly then $\RMST(\infty) = \EE[Y]$ by @eq-darth.
+Whereas @eq-darth represents the average survival over $[0,\infty)$, @eq-rmst represents the average survival time up to $\tau$.
+Equivalently, the RMST is the average amount of time each individual spends without experiencing the event up to $\tau$.
+The RMST treats all events happening after $\tau$ as if they happened at $\tau$.
+Hence, it is a truncated expectation, estimating: $\EE[\min(Y, \tau)]$
+
+The RMST can have several uses when comparing groups but here its relevant for its ability to give a meaningful lower-bound estimate of $\EE[Y]$ in the presence of censoring.
+For example, say individuals are observed over times $[0,100]$ and administrative censoring is present in the data so that $\hatS(\infty) > 0$ and $\EE[Y]$ cannot be reliably computed.
+However, one can compute $\RMST(100)$ which gives a lower-bound estimate for the expected survival: $\RMST(100) \leq \EE[Y]$.
+This avoids assumptions beyond $\tau=100$ and offers an interpretable prediction: "the average survival time is at least $\RMST(100)$" (@fig-survtsk-rmst).
```

Comment:
I've never seen this interpretation as lower-bound in the literature. Is that common?
I often read "the expected survival time within \tau years is RMST(\tau)

Reply:
I don't remember if I saw it in a particular article but it's true by definition, you're integrating up to \tau where \tau is either greater than the true event time (in which case it's identical to E[Y] ) or less than the true event time (in which case RMST < E[Y]). For me at least the lower-bound interpretation makes it a much more practical metric